docs/API.md

```diff
@@ -482,9 +479,10 @@ Example response:
 
 #### `"params"`
 
-An object with the property:
+An object with the properties:
 
-`"hostname"`: A *domain name*, required. The hostname whose IP addresses are to be resolved.
+* `"hostname"`: A *domain name*, required. The hostname whose IP addresses are to be resolved.
+* `"language"`: A [Language Tag], optional, used for validation error messages translation.
```

Comment:
Could you also link to where the default value is defined?

There are more of these that I haven't marked up.